### ISSUE TITLE
License Requirements?

### ISSUE THREAD

**Issue by jonwbstr** [MicrosoftDocs/OfficeDocs-WorkplaceAnalytics#104]
Hello, this article says:
>Workplace Analytics is licensed as an add-on to existing Office 365 subscriptions. Licenses are $2/license per month for Office 365 Enterprise E5 subscriptions and $6/license per month for all other qualifying Office 365 subscriptions.

We have O365 E5 and are unable to locate this "add-on"

[This Blog](https://www.microsoft.com/en-us/microsoft-365/blog/2017/07/05/transform-your-organization-with-microsoft-workplace-analytics/) says,
> Workplace Analytics is available now as an add-on to any Office 365 enterprise plan

However, we find no such add-on available for our O365 E5 licenses, and are unable to locate a demo tenant at demos.microsoft.com with the appropriate licenses either.

We are also a CSP Partner with microsoft, and are unable to locate a license for this product in the CSP catalog.

Thoughts?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d2f31d04-cc99-73e3-f057-647a895e4297
* Version Independent ID: b241adc5-0eba-64d8-eabd-cce6e31aac0c
* Content: [Environment and support aspects of Workplace Analytics - Workplace Intelligence](https://docs.microsoft.com/en-us/workplace-analytics/setup/environment-requirements)
* Content Source: [WorkplaceAnalytics/Setup/Environment-Requirements.md](https://github.com/MicrosoftDocs/OfficeDocs-WorkplaceAnalytics/blob/public/WorkplaceAnalytics/Setup/Environment-Requirements.md)
* Product: **wpa**
* GitHub Login: @madehmer
* Microsoft Alias: **v-midehm**

**Comment by paul9955** [comment]
Hello jonwbstr,
I looked into this and found something that might help. Today, Workplace Analytics is available only via an Office 365 Enterprise agreement. Since you're a CSP, the best route is to ask for help on this from the Workplace Analytics Partner team. You should be able to reach them through your usual Microsoft contact. 
Please comment back here if you need more help,
 - paul9955, Workplace Analytics documentation

**Comment by jonwbstr** [comment]
I also opened a ticket with o365 - ahem - Microsoft 365 support, and
learned the minimum seat requirement was lowered from 3000, but they were
not sure what the new minimum is. And, that it is part of the 365 E5
entitlements, provided we meet the seat requirement.

They pointed us to this page to request the feature be enabled for our
tenant
https://resources.office.com/ww-landing-workplace-analytics.html?LCID=EN-US

I will ask about the options for our clients, as it sounds like this may
not be something available through a CSP.

Thank you for the follow up, I am ready to close this request. I don't have
a strong opinion on what if any updates should be made to this article
based on what I've learned so far.

Regards,
-jon
